actionpack/lib/action_dispatch/system_testing/browser.rb

```diff
@@ -29,20 +29,27 @@ def options
         end
       end
 
+      def driver_option
+        @option ||= case type
+                    when :chrome
+                      Selenium::WebDriver::Chrome::Options.new
+                    when :firefox
+                      Selenium::WebDriver::Firefox::Options.new
+        end
+      end
```

Comment:
This should be private as it's only used by `headless_chrome_browser_options` and `headless_firefox_browser_options`.